### DISCUSSION TITLE
configuration file .itol.txt and How to get the tbprofiler tree?

### DISCUSSION THREAD

**Discussion by yustinus1** [jodyphelan/TBProfiler#199] [Q&A]
Hello Jody, 
I have run the script "tb-profiler collate" and got some .itol.text files. How to configure the files for phylogenetic construction? thank you

**Comment by jodyphelan** [comment]
Hi @yustinus1 

The phylogenetic reconstruction is a bit more complex. At the moment TB-Profiler does not provide any functionality to actually create the tree. The config files are generated, however the tree must be created with other software. It is planned for future releases, however it might take some time to be fully developed. I would suggest alternative software for the phylogenetic reconstruciton step (such as [snippy](https://github.com/tseemann/snippy) + [iqtree](http://www.iqtree.org/)).

I've gotten a few requests about this so I might try develop a simple phylogeny building function in the future versions.

**Reply by yustinus1** [reply]
Thanks Jody for your response and I can understand it now. Thank you also for the software recommendations that you recommend. But I want to ask, what genes are most relevant for phylogenetic construction?

**Reply by jodyphelan** [reply]
Sorry I missed your reply to this. To get as close to the 'true' phylogeny you should use all genes (minus the [low confidence regions](https://github.com/COMBAT-TB/tb_variant_filter/blob/main/bedfiles/farhat_rlc.bed)). Especially if you have WGS data available, you should not restrict to a subset of genes.
